README.md

```diff
@@ -6,7 +6,6 @@ vent
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/792bc7e54645427581da66cd6847cc31)](https://www.codacy.com/app/clewis/vent?utm_source=github.com&utm_medium=referral&utm_content=CyberReboot/vent&utm_campaign=badger)
 [![Build Status](https://travis-ci.org/CyberReboot/vent.svg?branch=master)](https://travis-ci.org/CyberReboot/vent)
 [![Documentation Status](https://readthedocs.org/projects/vent/badge/?version=latest)](http://vent.readthedocs.io/en/latest/?badge=latest)
-[![PyPI version](https://badge.fury.io/py/vent.svg)](https://badge.fury.io/py/vent)
```

Comment:
we still want the version badge, the one to be removed is actually called `GitHub Release Downloads`, sorry for the confusion there!

Reply:
I'll fix it.

Reply:
Did the link previously automatically start the .iso download? Is that why it is broken?